container-guide/concepts/containers-tools.xml

```diff
@@ -0,0 +1,207 @@
+      <link xlink:href="https://documentation.suse.com/sles/15-SP4/single-html/SLES-rmt/index.html#sec-rmt-deploy-kubernetes">&rmt;
+      documentation</link> for more information).
+    </para>
+  </section>
+  <section xml:id="sec-tools-distribution">
+    <title>Distribution</title>
+    <para>
+      <link xlink:href="https://github.com/distribution/distribution">Distribution</link>
+      is an open-source registry implementation for storing and distributing
+      container images using the OCI Distribution Specification. It provides a
+      simple, secure and scalable base for building a large scale registry
+      solution or running a simple private registry. Distribution can also
+      mirror &dhub; but
+      <link xlink:href="https://docs.docker.com/registry/recipes/mirror/#gotcha">not
+      any other private registry</link>.
+    </para>
+  </section>
+  <section xml:id="sec-tools-obs">
+    <title>Open Build Service</title>
+    <para>
+      The <link xlink:href="http://openbuildservice.org">Open Build Service
+      (OBS)</link> provides free infrastructure for building and storing RPM
+      packages including different container formats. The
+      <link xlink:href="https://registry.opensuse.org">OBS Container
+      Registry</link> provides a detailed listing of all container images built
+      by the OBS, complete with commands for pulling the images into your local
+      environment. The OBS
+      <link xlink:href="https://build.opensuse.org/image_templates">openSUSEcontainer
+      image templates</link> can be modified to specific needs, which offers
+      the easiest way to create your own container branch. Container images can
+      be built with &docker; tools from an existing image using a
+      <filename>Dockerfile</filename>. Alternatively, images can be built from
+      scratch using the &kiwi; image-building solution.
+    </para>
+    <para>
+      Instructions on how to build images on OBS can be found
+      <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/">here</link>.
+    </para>
+    <para>
+      &suse; Container Images, called &bci;s
+      (&bcia;s) are the only official container images. They are not available at
+      <link xlink:href="https://build.opensuse.org">https://build.opensuse.org</link>,
+      and the RPMs exported there are not identical to the internal ones. This
+      means that it is not possible to build officially supported images at
+      <link xlink:href="https://build.opensuse.org">https://build.opensuse.org</link>.
+    </para>
+    <para>
+      For more information about &bcia;s, refer to
+      <xref linkend="container-bci-general"/>
+      .
+    </para>
+  </section>
+  <section xml:id="sec-tools-kiwi">
+    <title>&kiwi;</title>
+    <para>
+      <link xlink:href="https://github.com/OSInside/kiwi">&kiwi;</link> Next
+      Generation is a multi-purpose tool for building images. In addition to
```

Comment:
Hmn, the current version of KIWI is already "KIWI NG". Maybe adapt the entity and remove the "Next Generation"?

Reply:
Fixed in openSUSE/doc-kit#70